modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,228 @@
+
+== Limitations
+
+* You can only enable Iceberg topic integration for new Redpanda topics. 
+* It is not possible to append or backfill data from Redpanda topics to an existing Iceberg table.
+* JSON schemas are not currently supported. For Avro schemas, records cannot contain fields greater than 128 KB.
+* You can only use one schema per topic. Schema versioning as well as upcasting (where a value is cast into its more generic data type) are not supported. 
+
+== Architecture 
+
+Apache Iceberg is an open source format specification for defining structured tables in a data lake. The table format lets you quickly and easily manage, query, and process huge amounts of structured and unstructured data. This is similar to the way in which you would manage and run SQL queries against relational data in a database or data warehouse. The open format lets you use many different languages, tools, and applications to process the same data in a consistent way, so you can avoid vendor lock-in. This data management system is referred to as a _data lakehouse_.
+
+In the Iceberg specification, tables comprise the following:
+
+* Data layer
+** Data files: Store the data. The Iceberg integration currently supports the Parquet file format. Parquet files are column-based and suitable for analytical workloads at scale. They come with compression capabilities that optimize files for object storage. 
+* Metadata layer
++
+--
+** Manifest files: Track data files and contain metadata about these files, such as record count, partition membership, and file paths.
+** Manifest list: Tracks all the manifest files belonging to a table, including file paths and upper and lower bounds for partition fields. 
+** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
+--
++
+In the Redpanda Iceberg integration, the manifest files are in JSON format.
+* Catalog: Contains the current metadata pointer for the table. Clients reading and writing data to the table see the same version of the current state of the table. You'll configure your Iceberg catalog to point to your object storage bucket or container where the Redpanda data in Iceberg format is located. Redpanda uses the https://iceberg.apache.org/concepts/catalog/#catalog-implementations[Iceberg REST catalog^] endpoint to update your catalog when there are changes to the Iceberg data and metadata.
+
+image::shared:iceberg-integration.png[]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage. The brokers also write events to segment files stored on disk. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through compatible business intelligence and machine learning tools. 
```

Comment:
maybe  'many compatible Data Lakehouse, Machine Learning, Data Engineering and Business Intelligence Tools' and link to https://iceberg.apache.org/vendors/